internal/pkg/archer/app.go

```diff
@@ -31,3 +31,13 @@ type ApplicationGetter interface {
 type ApplicationCreator interface {
 	CreateApplication(app *Application) error
 }
+
+const (
+	// AppCfnTemplateNameFormat is the base output file name when `app package`
+	// is called. This is also used to render the pipeline CFN template.
+	AppCfnTemplateNameFormat = "%s.stack.yml"
+	// AppCfnTemplateConfigurationNameFormat is the base output configuration
+	// file name when `app package` is called. It's also used to render the
+	// pipeline CFN template.
+	AppCfnTemplateConfigurationNameFormat = "%s-%s.params.json"
```

Comment:
What do you think about moving this to the `cloudformation` pkg?

Reply:
I realize that'd introduce a circular dependency with `PipelineStage` but maybe those structs should be in the `cloudformation` pkg as well since we don't have a story yet for how a pipeline looks like outside of cfn?

Reply:
let's hold off on moving thing/refactoring for now